testbed/compositions/rfc_k8s.toml

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name    = "rfcBBL103A"
+author  = "adlrocha"
+
+[global]
+plan    = "testbed"
+case    = "ipfs-transfer"
+builder = "docker:generic"
+runner  = "cluster:k8s"
+
+total_instances = 6
+
+[global.build_config]
+  push_registry=true
+  go_proxy_mode="remote"
+  go_proxy_url="http://localhost:8081"
+  registry_type="aws"
```

Comment:
No need for the `skip_runtime_image` here anymore, as we are using the `docker:generic` builder, and we specify a concrete Dockerfile for our test plan.